### PR TITLE
set up SingleUser component, updated user store

### PR DESCRIPTION
**Pull request by markallenblanchard**

### CODE REVIEW

File: client/components/SingleUser.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+			return (
+				<div>
+					<h1>User Not Found</h1>
+				</div>
+			);
+		}
+	}
+}
+
+const mapStateToProps = (state) => ({user: state.defaultUser});
```
Comment (mustafadane) [inline client/components/SingleUser.js]: I think this will be user: state.users.defaultUser

File: client/components/SingleUser.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
```
Comment (mustafadane) [inline client/components/SingleUser.js]: import { getUser } from store

File: client/components/SingleUser.js (hunk 1 above)
Comment (mustafadane) [reply client/components/SingleUser.js]: probably state.user only

File: client/components/SingleUser.js (hunk 2 above)
Comment (markallenblanchard) [reply client/components/SingleUser.js]: good catch

File: client/components/SingleUser.js (hunk 1 above)
Comment (markallenblanchard) [reply client/components/SingleUser.js]: We decided to build a new store separating Oauth user from  all users

**Pull request merged**